src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -88,6 +88,8 @@ endif()
 # newer CMake versions
 if (CONDUIT_DIR)
     axom_assert_is_directory(VARIABLE_NAME CONDUIT_DIR)
+
+    unset(CONDUIT_FOUND CACHE)
```

Comment:
@white238 and @cyrush -- I had to add this for incremental reconfigs since conduit's config files explicitly check for `CONDUIT_FOUND`, but `conduit::conduit` is not otherwise a target

Reply:
This is b/c axom is setting `CONDUIT_FOUND` a few lines down:

https://github.com/LLNL/axom/blob/75126d6a0b7d99541b017c48c628d24a571cdf6a/src/cmake/thirdparty/SetupAxomThirdParty.cmake#L102

Not sure 100% why, but maybe remove there b/c that seems to be what you are doing effectively by unsetting it before we call find again.

Reply:
Thanks -- I think it's there for consistency w/ how we treat other packages. 
@white238 mentioned offline that we might not need it to be a `CACHE` variable.

Reply:
Got it, but Conduit sets `CONDUIT_FOUND` itself when it is found -- so I think it could be dropped unless is some reason downstream in Axom's cmake that this needs to be a cached var?

Reply:
It shouldn't, my guess is I got a little excited to make everything "symmetrical" for lack of a better word.

Reply:
Are we ok w/ my proposed change? Or should I remove both (all) references to `CONDUIT_FOUND`  in `SetupAxomThirdParty.cmake`?

Reply:
Either is fine.

Reply:
Thanks -- in that case, I'll leave it as is.